core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5892,6 +5892,10 @@ components:
             - GCP
             - AZURE
             - UNKNOWN
+        landingZoneId:
```

Comment:
I suppose I should update orchestration's swagger too? https://firecloud-orchestration.dsde-dev.broadinstitute.org/#/BillingV2/listUserBillingProjectsV2